src/app/counter/counter/counter.component.ts

```diff
@@ -43,4 +48,9 @@ export class CounterComponent implements OnInit {
     document.getElementById("book").classList.remove("side-link-active");
     document.getElementById("cancel").classList.add("side-link-active");
   }
+
+  resetLinks() {
+    document.getElementById("book").classList.remove("side-link-active");
```

Comment:
Did this solve the issue of the sidebar remaining active?

Reply:
Nope

Reply:
Actually it does solve it when clicking the button to change the traveler, but the issue still occurs when clicking the Counter link in the top nav.